pkg/apis/kops/v1alpha1/componentconfig.go

```diff
@@ -417,3 +417,14 @@ type CloudConfiguration struct {
 	VSphereDatastore     *string `json:"vSphereDatastore,omitempty"`
 	VSphereCoreDNSServer *string `json:"vSphereCoreDNSServer,omitempty"`
 }
+
+// HasAdmissionController checks if a specific admission controller is enabled
+func (c *KubeAPIServerConfig) HasAdmissionController(name string) bool {
```

Comment:
I think to date we've avoided methods on the API objects themselves, but this one is simple enough that we can just see what happens :-)  I'm surprised we want it on anything other than the internal version (i.e. I'm surprised we want it on v1alpha1 and v1alpha2)